roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -9,8 +9,10 @@ openshift_logging_mux_namespace: logging
 
 ### Common settings
 openshift_logging_mux_nodeselector: "{{ openshift_hosted_logging_mux_nodeselector_label | default('') | map_from_pairs }}"
-openshift_logging_mux_cpu_limit: 500m
-openshift_logging_mux_memory_limit: 1Gi
+openshift_logging_mux_cpu_limit: "{{ openshift_hosted_logging_mux_cpu_limit | default('500m') }}"
```

Comment:
Thank you, Eric!  It was a bug from my misunderstanding.  I removed "_hosted" from the line(s) and it worked just as expected.  I'm committing the change.
openshift_logging_mux_memory_limit: "{{ openshift_logging_mux_memory_limit | default('1Gi') }}"